lib/membrane/video_compositor/scene/temporal.ex

```diff
@@ -0,0 +1,108 @@
+defmodule Membrane.VideoCompositor.Scene.Temporal do
```

Comment:
I think this would be better:
```suggestion
defmodule Membrane.VideoCompositor.TemporalScene do
```